Packs/CommonScripts/ReleaseNotes/1_7_28.md

```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
-##### GetDockerImageLatestTag
-- Updated the Docker image to: *demisto/python3:3.10.6.33415*.
+##### SetDateField
+- Updated the docker image to *demisto/python3:3.9.5.20070*.
```

Comment:
Why are we editing old rn?

Reply:
At the time I was working on this PR the version was 1.7.28. I'll regenerate them to fix this.